examples/icd_code_prediction/icd_coding.py

```diff
@@ -0,0 +1,86 @@
+import sys
+import yaml
+from termcolor import colored
+
+from forte.common.configuration import Config
+from forte.data.data_pack import DataPack
+from forte.data.readers import PlainTextReader
+from forte.pipeline import Pipeline
+from forte.processors.writers import PackIdJsonPackWriter
+
+from ft.onto.base_ontology import (
+    Token,
+)
+from fortex.spacy import SpacyProcessor
+
+from ftx.medical.clinical_ontology import MedicalArticle
+from fortex.health.readers.mimic3_note_reader import Mimic3DischargeNoteReader
+from fortex.health.processors.icd_coding_processor import (
+    ICDCodingProcessor,
+)
+
+
+def main(
+    input_path: str,  # Path to mimic3 data if use_mimic3_reader=True else path to notes directory
+    output_path: str,  # Path to output directory
+    max_packs: int = -1,  # Max number of notes to read from mimic3 dataset. Set to -1 to read all.
+    use_mimic3_reader: bool = True,  # Read from mimic3 dataset or plain text
+):
+    pl = Pipeline[DataPack]()
+
+    if use_mimic3_reader is False:
+        pl.set_reader(PlainTextReader())
+    else:
+        pl.set_reader(Mimic3DischargeNoteReader(), config={"max_num_notes": max_packs})
+
+    config = Config(yaml.safe_load(open("config.yml", "r")), None)
```

Comment:
This config loading is correct. Just as an example, it isn't really necessary. The config file is quite brief, we should probably directly initialize the config here, so that one can actually see what happens in one file.